src/main/java/edu/ksu/canvas/attendance/model/AttendanceSummaryModel.java

```diff
@@ -45,14 +48,16 @@ public Entry(long courseId, long sectionId, long studentId,
             this.totalClassesExcused = -1;
             this.totalClassesMissed = -1;
             this.totalClassesTardy = -1;
+            this.totalClassesPresent = -1;
         }
 
         public Entry(long courseId, long sectionId, long studentId,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 10 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00107)